cli/index.js

```diff
@@ -649,6 +649,18 @@ RainbowConfig.addOptions(
   ]
 )
 
+RainbowConfig.addOptions(
+  program
+    .command('eth-to-near-find-storage-proof <storage_data>')
+    .description('Get eth-to-near storage proof by storage data.'),
```

Comment:
Please add a description of the `storage_data` format. As I see it is a json with some fields.
Also I think it will be better if we pass it not like a json, but just like arguments `contractAddress, slotKey, blockNumber`

Reply:
Fixed in 064947f3221b1f1031346be2535bb7fc0e775726